stage1/unit4/objects_and_functions.js

```diff
@@ -0,0 +1,59 @@
+//   Создать объект car, добавить к нему свойство color со значением 'черный'
+let car = {
+    color: 'black',
+    carPower: 245,
+}
+
+//Изменить свойство color объекта car на 'зеленый'
+car.color = 'green'
+
+// В объект car добавить свойство power, которое является функцией и выводит в консоль мощность двигателя
+car.power = function () {
+     return console.log('Лошадиных сил: ' + this.carPower)
+}
+
+car.power()
+
+// На склад принимают груши и яблоки, напишите функцию, которая возвращает результат сложения количества принятых груш и яблок
+const appleAmount = 2567
+const pearAmount = 1452
+
+function amountOfFruitsReceived(appleAmount, pearAmount) {
+    return +appleAmount + +pearAmount
+}
+
+console.log('Всего приняли груш и яблок: ' + amountOfFruitsReceived(appleAmount, pearAmount))
+
+// В терминале оплаты сохранено ваше имя, напишите функцию для определения имени в терминале(если вы ввели ваше имя, то привет + имя, если нет, то нет такого имени)
+const userName = 'Gleb'
+
+function verifyUser(userInputName = 'Имя не введено') {
```

Comment:
Default value for parameter is redundant here.

Reply:
agreed. redundant code has been deleted